packages/node-core/src/indexer/mmrMigrate.service.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020-2022 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import {existsSync} from 'fs';
+import {DEFAULT_WORD_SIZE} from '@subql/common';
+import {NodeConfig, getExistingProjectSchema, getLogger, PgBasedMMRDB} from '@subql/node-core';
+import {FileBasedDb} from '@subql/x-merkle-mountain-range';
+import {Logging, Sequelize} from 'sequelize';
+
+const logger = getLogger('mmr-migrate');
+
+const DEFAULT_DB_SCHEMA = 'public';
```

Comment:
https://github.com/subquery/subql/pull/1648

We just deprecated localMode, user can always save this in configured schema